bin/faucet/examples/typescript/request_tokens.ts

```diff
@@ -0,0 +1,95 @@
+import { sha3_256 } from '@noble/hashes/sha3';
+import fs from 'fs';
+
+async function sendPowRequest(baseUrl: string, accountId: string) {
```

Comment:
For this and other functions, could we specify the return type? (otherwise, it is not immediately clear which functions return values and which ones do not).